lib/beaker/hypervisor/openstack.rb

```diff
@@ -220,16 +220,15 @@ def provision
 
       @hosts.each do |host|
         ip = get_ip
-        host[:vmhostname] = ip.ip.gsub '.','-'
-        host[:vmfqdn] = host[:vmhostname] + '.rfc1918.puppetlabs.net'
```

Comment:
it looks like the last person to set this code to use the `:vmfqdn` was @spjmurray in [this change](https://github.com/puppetlabs/beaker/commit/f41fa414ac7f397b7f140db144b40048427aac6e).

@spjmurray, I notice that in your description, you talk about how your commit attaches both a hostname and fqdn to a host and that fqdn is needed for backwards compatibility. Is this openstack backwards compatibility?